### PR TITLE
TF-2533 Create ADR to put memory footprint task on hold

### PR DESCRIPTION
**Pull request by tddang-linagora**
### Related issue
- #2533

### CODE REVIEW

File: docs/adr/0044-memory-leak-at-current-state.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - Email content view:
+    - Web: Resolved
+    - Mobile: Remained until [InAppWebView memory leak on Android #1023](https://github.com/pichillilorenzo/flutter_inappwebview/issues/1023) is fixed
+  - Identity create/edit:
+    - Web: Resolved
+    - Mobile: No problem detected, nothing changed.
+
+## Decision
+
+- This work will be on hold until further notice
```
Comment (chibenwa) [inline docs/adr/0044-memory-leak-at-current-state.md:30]: On which conditions would work on this be resumed?

Comment (tddang-linagora) [reply docs/adr/0044-memory-leak-at-current-state.md:30]: On @hoangdat command

Comment (chibenwa) [reply docs/adr/0044-memory-leak-at-current-state.md:30]: I suggest: if significant memory leaks are reportedeg consuming twice the regular amount of memory within one day usage.

Comment (tddang-linagora) [reply docs/adr/0044-memory-leak-at-current-state.md]: @chibenwa Done

**Review by hoangdat — APPROVED** [review]

**Pull request merged**